src/main/scala/scalaExercises.scala

```diff
@@ -1,14 +1,65 @@
-class scalaExercises
-{
+import scala.math
+object scalaExercises {
+  def checker() {
+    for (i <- 100000000 to 300000000 if primechecker(i) == true) {
```

Comment:
primechecker(i) returns a boolean by itself, therefore you could simplify this to 

for (i <- 100000000 to 300000000 if primechecker(i)) {